src/composites/FeaturesContainer/Overview/OverviewCollector.ts

```diff
@@ -0,0 +1,103 @@
+import { overviewData } from './index.ts';
+import { State } from "../../../state.ts";
+import { useAtomValue } from "jotai";
+
+//expected DSStats for compact-output: 0-0-0-0-6 / 1-1-0-0-34 / 4-0-1-0-161
+
+export function OverviewDataCollector():overviewData {
+    const dataset = useAtomValue(State.dataset);
```

Comment:
this is not a React component right? Let's change this function to explicitly accept the data as an argument, and remove the useAtomValue

React hooks (such as useAtomValue) cannot be used outside of components. This probably only works because you're using this directly in some component body, but this opens up room for bugs in the future if someone uses this outside of a component.

Reply:
If you want to keep using the hook here, rename the function to `useXXXX` to clarify that it must be treated like a hook.

Reply:
Data now passed explicitly